packages/epo-react-lib/src/atomic/Video/index.tsx

```diff
@@ -0,0 +1,3 @@
+export { default } from "./patterns/YouTube";
```

Comment:
export different versions of the video player with the default being YouTube, for backwards compatibility reasons.